addons/im_livechat/models/discuss_channel.py

```diff
@@ -469,3 +469,10 @@ def _action_unfollow(self, partner=None, guest=None, post_leave_message=True):
             # sudo: discuss.channel - last operator left the conversation, state must be updated.
             channel_sudo.livechat_active = False
             self._bus_send_store(self, "livechat_active")
+
+    def _add_members(self, **kwargs):
+        if isinstance(kwargs.get("inviting_partner"), int):
+            kwargs["inviting_partner"] = self.env["res.partner"].browse(kwargs["inviting_partner"])
+        if isinstance(kwargs.get("users"), int):
+            kwargs["users"] = self.env["res.users"].browse(kwargs["users"])
+        return super()._add_members(**kwargs)
```

Comment:
it's not acceptable to add such an override for demo data purpose. you can try to use ` eval="obj().search` in your xml.